package.json

```diff
@@ -102,6 +102,7 @@
     "rxjs": "^6.5.5",
     "semver": "^6.3.0",
     "ts-jest": "^26.4.4",
+    "tslib": "2.4.0",
```

Comment:
Why do we need this `tslib` package? We didn't need it thus far, I want to know what dependency made this a requirement, and if it has to sit in the root package.json.